src/Auth0.Core/Exceptions/ApiError.cs

```diff
@@ -29,6 +30,12 @@ public class ApiError
         [JsonProperty("message")]
         public string Message { get; set; }
 
+        /// <summary>
+        /// Additional key/values that might be returned by the error such as `mfa_required`.
+        /// </summary>
+        [JsonProperty("extraData")]
+        public Dictionary<string, string> ExtraData { get; set; } = new Dictionary<string, string>();
```

Comment:
I'm not entirely convinced of the name of the property/method "ExtraData". But I can't come up with something else right now.